app/components/uploadlaunchoverlay/uploadlaunchoverlay.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2017, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ */
+
+import _ from 'lodash';
+import React, { Component } from 'react';
+import cx from 'classnames';
+import GitHub from 'github-api';
+import ModalOverlay from '../modaloverlay';
+import { URL_UPLOADER_DOWNLOAD_PAGE } from '../../core/constants';
+import logoSrc from '../uploaderbutton/images/T-logo-dark-512x512.png';
+
+const github = new GitHub();
+
+class UploadLaunchOverlay extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      latestWinRelease: null,
+      latestMacRelease: null,
+      error: null
```

Comment:
But I do see that they're part of Airbnb's eslint, so in the interest of moving that direction, I'll tack them in